lib/common/messages.c

```diff
@@ -221,6 +221,10 @@ pcmk__process_request(pcmk__request_t *request, GHashTable *handlers)
     }
 
     handler = g_hash_table_lookup(handlers, request->op);
+    if (handler == NULL) {
+        return NULL;
+    }
+#if 0
```

Comment:
Could we just wait until the end to add the default handler (`handle_unknown_request`)? If the default handler is `NULL`, then we get the same result when we look up `""` in the hash table, without having to do this `#if 0` guard. The only difference is that we log an info message.

Reply:
Yes, that is a much better way of doing it.